From: Sebastian Kuzminsky <seb@DesktopMetal.com>
Date: Tue, 22 Jan 2019 12:53:51 -0700
Subject: install dm-gcode-generator resources

This overwrites the resources from Cura 3.2, but it can't be helped.
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 49c5148..34229c5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -234,7 +234,7 @@ endif()
 add_custom_command(TARGET CuraEngine POST_BUILD
                    COMMENT "copying resources to build dir"
                    COMMAND ${CMAKE_COMMAND} -E copy_directory
-                       ${CMAKE_SOURCE_DIR}/resources $<TARGET_FILE_DIR:CuraEngine>)
+                       ${CMAKE_SOURCE_DIR}/resources $<TARGET_FILE_DIR:CuraEngine>/resources)
 
 set(CURA_ENGINE_VERSION_NUMBER "3.2.1" CACHE STRING "Version number of CuraEngine")
 set_target_properties(_CuraEngine PROPERTIES COMPILE_DEFINITIONS "VERSION_NUMBER=\"${CURA_ENGINE_VERSION_NUMBER}\";VERSION=\"${CURA_ENGINE_VERSION}\"")
@@ -242,4 +242,5 @@ set_target_properties(_CuraEngine PROPERTIES COMPILE_DEFINITIONS "VERSION_NUMBER
 # Installing CuraEngine.
 include(GNUInstallDirs)
 install(TARGETS CuraEngine DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_PREFIX}/share/cura)
 include(CPackConfig.cmake)
